astropy/units/format/console.py

```diff
@@ -67,7 +71,7 @@ def to_string(cls, unit, inline=True):
             if unit.scale == 1:
                 s = ""
             else:
-                s = cls.format_exponential_notation(unit.scale)
+                s = cls.format_exponential_notation(unit.scale) + " "
```

Comment:
To be nitpicky here: for a scale-only unit this adds a trailing space:
```python
>>> u.Unit(10).to_string('unicode')
'10 '
```
Probably one again use a "parts" list and `" ".join(parts)` at the end, as in `generic.py` (but just with 1-2 parts)?

Reply:
Good catch! the parts use is a bit tricky given the possible multiple lines so I just added the space only when needed. It turned out for `latex` the same mistake was made, so I corrected that while I was at it.